estimate_gain: 5